turbinia/turbiniactl.py

```diff
@@ -30,7 +30,6 @@
 from turbinia import config
 from turbinia import TurbiniaException
 from turbinia.config import logger
-from turbinia.lib import google_cloud
```

Comment:
This gets called here https://github.com/google/turbinia/blob/08ce6c7ea6d9be68f478d5db1b4605f0dea1a1f7/turbinia/turbiniactl.py#L455. We should keep it or move it to the if statement